daemon/cmd/daemon_main.go

```diff
@@ -510,6 +510,9 @@ func init() {
 	flags.String(option.NodePortAcceleration, option.NodePortAccelerationNone, "BPF NodePort acceleration via XDP (\"native\", \"none\")")
 	option.BindEnv(option.NodePortAcceleration)
 
+	flags.Bool(option.EnableSessionAffinity, false, "Enable support for service sessionAffinity")
```

Comment:
`s/sessionAffinity/Session Affinity/`

Reply:
(changed to `session affinity` as k8s docs do not capitalize it)